dll/win32/devmgr/lang/ro-RO.rc

```diff
@@ -298,7 +299,7 @@ BEGIN
 END
 STRINGTABLE DISCARDABLE
 BEGIN
-  IDS_MENU_UPDATE           "Actualizare modúl-pilot…"
+  IDS_MENU_UPDATE           "Actualizează modulul-pilot…"
```

Comment:
The same as above regarding the ellipsis-ending cases. Either revert or take note for yourself to spread this change across the rest of the codebase.

Reply:
I did the changes you suggested to me.

Reply:
You seem to have decided on keeping long infinitive for the intermediary actions. Now all that remains is for you to remain consistent about it (look above, where you still left „A&ctualizează modulul-pilot…”).

Reply:
Yeah, I forgot about it.